app.js

```diff
@@ -4,16 +4,20 @@ const app = new Koa();
 
 const body = require('koa-json-body');
 const cors = require('@koa/cors');
+const nearAPI = require('near-api-js');
 
 const constants = require('./constants');
+const recaptchaValidator = require('./RecaptchaValidator');
 
 const { RECOVERY_METHOD_KINDS, SERVER_EVENTS } = constants;
 
 app.use(require('koa-logger')());
 app.use(body({ limit: '500kb', fallback: true }));
 app.use(cors({ credentials: true }));
 
-let reportException = () => {};
```

Comment:
nit: let's try to not reformat otherwise untouched code (slightly increases review burden).

Reply:
Will do! Also, FYI, you can hide purely whitespace changes from the diff entirely in GH interface
![image](https://user-images.githubusercontent.com/29758232/116215435-88ade800-a6fc-11eb-94ff-ccce9a7f1f50.png)